pkg/common/git.go

```diff
@@ -0,0 +1,204 @@
+package common
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/nuclio/nuclio/pkg/cmdrunner"
+
+	"github.com/nuclio/errors"
+	"github.com/nuclio/logger"
+	"gopkg.in/src-d/go-git.v4"
+	"gopkg.in/src-d/go-git.v4/plumbing"
+	githttp "gopkg.in/src-d/go-git.v4/plumbing/transport/http"
+)
+
+type GitAttributes struct {
```

Comment:
there's too many `Git` prefix here, I'd suggest moving it to its own package, with `types.go`